src/studio/src/designer/backend/Services/Implementation/RepositorySI.cs

```diff
@@ -1213,6 +1214,21 @@ public async Task<RepositoryClient.Model.Repository> CopyRepository(string org,
             CommitInfo commitInfo = new CommitInfo() { Org = org, Repository = targetRepository, Message = $"App cloned from {sourceRepository} {DateTime.Now.Date.ToShortDateString()}" };
             _sourceControl.PushChangesForRepository(commitInfo);
 
+            // Final changes are made in a seperate branch to be reviewed by developer
```

Comment:
Although this method got a bit big, I like the readability of the calls you added below here. All on the same level and good method names = easy reading:)